lib/app/components/layouts/collapsing_header_scroll_links_layout.dart

```diff
@@ -0,0 +1,231 @@
+// SPDX-License-Identifier: ice License 1.0
+
+import 'package:flutter/material.dart';
+import 'package:flutter_hooks/flutter_hooks.dart';
+import 'package:go_router/go_router.dart';
+import 'package:ion/app/components/scroll_to_top_wrapper/scroll_to_top_wrapper.dart';
+import 'package:ion/app/components/section_separator/section_separator.dart';
+import 'package:ion/app/components/tabs_header/scroll_links_tabs_header.dart';
+import 'package:ion/app/extensions/extensions.dart';
+import 'package:ion/app/features/user/model/tab_type_interface.dart';
+import 'package:ion/app/hooks/use_scroll_top_on_tab_press.dart';
+import 'package:ion/app/router/components/navigation_app_bar/navigation_app_bar.dart';
+import 'package:ion/app/router/components/navigation_app_bar/navigation_back_button.dart';
+import 'package:ion/generated/assets.gen.dart';
+
+const _kExpandedHeaderContentOffset = -30.0;
+const _kTabBarHeight = 48.0;
```

Comment:
we're not using `k` prefixes for constants anywhere, maybe for consistency do not start doing so?